Lib/test/test_os.py

```diff
@@ -70,6 +64,8 @@
 from test.support.os_helper import FakePath
 
 
+asyncio.set_event_loop_policy(asyncio.DefaultEventLoopPolicy())
+
```

Comment:
Please add the following to prevent a warning about unexpected environment modification.
```
def tearDownModule():
    asyncio.set_event_loop_policy(None)
```

Reply:
Thank you, changed.